src/BassetServiceProvider.php

```diff
@@ -160,6 +164,29 @@ public function terminate(): void
         $basset->cacheMap->save();
     }
 
+    /**
+     * Loads needed basset disks.
+     *
+     * @return void
+     */
+    public function loadDisk(): void
```

Comment:
I thought we talked about adding the disk only if `config('backpack.basset.disk') == 'basset'`. Did we forget or decide against that?

Reply:
I'm ok with whatever you guys decide on this. It just stood out to me.

Reply:
We do it inside the function. Line 174 of this file.

Reply:
Not quite. From what I understand, what we do here is "_if there is not `basset` disk already set up, set it up_". That is perfectly fine to to.

But what I remember we talked about, before the above, was also "_if nobody has configure Basset to use the `basset` disk, don't even set up a `basset` disk_". Basically to only set up a basset disk if the dev has chosen it. It might be overkill. But I remember it's what we decided at the time.

Reply:
Right! I overlooked that. 

Just added the condition we talked about.